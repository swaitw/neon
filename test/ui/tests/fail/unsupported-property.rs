#[neon::export(foo)]
static STRING: &str = "";

#[neon::export(foo)]
fn unsupported() {}

fn main() {}
