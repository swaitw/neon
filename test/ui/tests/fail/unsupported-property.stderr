error: unsupported property
 --> tests/fail/unsupported-property.rs:1:16
  |
1 | #[neon::export(foo)]
  |                ^^^

error: unsupported property
 --> tests/fail/unsupported-property.rs:4:16
  |
4 | #[neon::export(foo)]
  |                ^^^
